sia/peers.go

```diff
@@ -14,11 +14,26 @@ func (c *Core) initializeNetwork(addr string, nobootstrap bool) (err error) {
 		return
 	}
 
-	c.server.Register("AcceptBlock", c.AcceptBlock)
-	c.server.Register("AcceptTransaction", c.AcceptTransaction)
-	c.server.Register("SendBlocks", c.SendBlocks)
-	// c.server.Register("NegotiateContract", c.NegotiateContract)
-	// c.server.Register("RetrieveFile", c.RetrieveFile)
+	err = c.server.RegisterRPC("AcceptBlock", c.AcceptBlock)
+	if err != nil {
+		return
+	}
+	err = c.server.RegisterRPC("AcceptTransaction", c.AcceptTransaction)
+	if err != nil {
+		return
+	}
+	err = c.server.RegisterRPC("SendBlocks", c.SendBlocks)
+	if err != nil {
+		return
+	}
+	c.server.RegisterRPC("NegotiateContract", c.host.NegotiateContract)
+	if err != nil {
+		return
+	}
+	c.server.RegisterRPC("RetrieveFile", c.host.RetrieveFile)
+	if err != nil {
+		return
+	}
```

Comment:
I still don't like this. A non-nil error here is unrecoverable, and an error doesn't help the user any more than a panic would. As I've said multiple times, the only way for a user to see such an error is if the `RegisterRPC` call has *never been run even a single time* before being shipped. 
Plus the last two errors aren't even checked.